xar-assembly.xml

```diff
@@ -49,6 +49,16 @@
             <artifactId>${project.artifactId}</artifactId>
             <version>${project.version}</version>
         </dependencySet>
+        <dependencySet>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.9.8</version>
```

Comment:
As you have the jackson version repeated in 4 places, it might be sensible to define it once as a property.